src/scripts/actions/UserActions.js

```diff
@@ -117,9 +117,9 @@ export const setupKeystore = () => async (
 
   // Case 2: we have a secured wallet is localStorage
   if (walletStringSecure && ACTIVATE_SECURE_WALLET) {
-    console.log('Try to open encrypted keystore')
+    // console.log('Try to open encrypted keystore')
     // Need to ask the PIN
-    dispatch(openModal(MODAL.ASK_PASSWORD))
+    // dispatch(openModal(MODAL.ASK_PASSWORD))
```

Comment:
remove commented lines?

Reply:
it's temporary, because now i've removed the `askPassword` at the beginning but maybe it will come back|

Reply:
I would just delete it, we can easily add it back in later